### ISSUE TITLE
Create example(s) to demonstrate all video modes

### ISSUE THREAD

**Issue by QuzarDC** [KallistiOS/KallistiOS#441]
KOS currently supports a number of video modes that have no examples that demonstrate their use. Among them 256x256 and 720x480. With the long-time misconceptions around 800x608, (and the newer misconceptions around Moop's HD modes) it would be useful to ensure there are examples that can demonstrate all of our video modes (especially the ones above 640x480).

I may take this up myself, but wanted to make sure to put in an issue to record the gap. To my mind the examples should also clearly be able to demonstrate if the mode is not working correctly, so perhaps rendering reference test patterns rather than just an xor pattern or a texture that gets stretched.

**Comment by gyrovorbis** [comment]
@QuzarDC didn't this actually just get checked in as part of your video mode refactor endeavor?

**Comment by QuzarDC** [comment]
No, that was merely a demonstration of multi-buffering since we had no example that used it.
